src/components/organism/WithdrawPoolLiquidity/index.tsx

```diff
@@ -0,0 +1,357 @@
+        <BackArrow width={24} height={24} />
+      </button>
+      <h3 className="heading-6 font-unbounded-variable font-normal">
+        {location?.state?.pageType === LiquidityPageType.removeLiquidity
+          ? t("poolsPage.removeLiquidity")
+          : t("poolsPage.addLiquidity")}
+      </h3>
+      <hr className="mb-0.5 mt-1 w-full border-[0.7px] border-b-modal-header-border-color" />
+      <TokenAmountInput
+        tokenText={selectedTokenA?.nativeTokenSymbol}
+        labelText={t("poolsPage.withdrawalAmount")}
+        tokenIcon={<DotToken />}
+        tokenValue={selectedTokenNativeValue?.tokenValue}
+        onClick={() => null}
+        onSetTokenValue={() => null}
+        selectDisabled={true}
+        disabled={true}
+      />
+      <TokenAmountInput
+        tokenText={selectedTokenB?.tokenSymbol}
+        labelText={t("poolsPage.withdrawalAmount")}
+        tokenIcon={<DotToken />}
+        tokenValue={selectedTokenAssetValue?.tokenValue}
+        onClick={() => setIsModalOpen(true)}
+        onSetTokenValue={() => null}
+        selectDisabled={true}
+        disabled={true}
+      />
+      <div className="mt-1 text-small">{transferGasFeesMessage}</div>
+
+      <div className="flex w-full flex-col gap-2 rounded-lg bg-purple-50 px-4 py-6">
+        <div className="flex w-full justify-between text-medium font-normal text-text-color-label-light">
+          <div className="flex">{t("tokenAmountInput.slippageTolerance")}</div>
+          <span>15%</span>
+        </div>
+        <div className="flex w-full gap-2">
+          <div className="flex w-full basis-8/12 rounded-xl bg-white p-1 text-large font-normal text-text-color-header-light">
+            <button
+              className={`flex basis-1/2 justify-center rounded-lg  px-4 py-3 ${
+                slippageAuto ? "bg-purple-100" : "bg-white"
+              }`}
+              onClick={() => {
+                setSlippageAuto(true);
+                setSlippageValue(15);
+              }}
+            >
+              {t("tokenAmountInput.auto")}
+            </button>
+            <button
+              className={`flex basis-1/2 justify-center rounded-lg px-4 py-3 ${
+                slippageAuto ? "bg-white" : "bg-purple-100"
+              }`}
+              onClick={() => setSlippageAuto(false)}
+            >
+              {t("tokenAmountInput.custom")}
+            </button>
+          </div>
+          <div className="flex basis-1/3">
+            <div className="relative flex">
+              <NumericFormat
+                value={slippageValue}
+                onValueChange={({ floatValue }) => setSlippageValue(floatValue)}
+                fixedDecimalScale={true}
+                thousandSeparator={false}
+                allowNegative={false}
+                className="w-full rounded-lg bg-purple-100 p-2 text-large  text-text-color-label-light outline-none"
+                placeholder="15"
+                disabled={slippageAuto ? true : false}
```

Comment:
Please try to avoid this style true/false, you can write it something like this: !!slippageAuto.